Misc/NEWS.d/next/Core and Builtins/2022-02-11-18-51-11.bpo-39355.rcUPdT.rst

```diff
@@ -0,0 +1 @@
+Make Python.h compatible with C++20 compilers.
```

Comment:
Would you mind to explain that the change is to avoid the usage of "module" which became a keyword in C++20?

Reply:
Yes @vstinner, you are right!

I have mentioned in bugs.python.org, about this change. 
After making a little research, i realized that this "module" keyword has meaning with "export"
Sorry i am not c++ expert )) If somebody have knowledge about it, share it plz :)

Reply:
I don't understand your comment. Do you mean that https://bugs.python.org/issue39355 is not a bug and the Python C API is compatible with C++20?

If there is an issue, please explain the change in NEWS entry. "Make Python.h compatible with C++20 compilers." is not enough.

Reply:
@vstinner I asked question ))

Reply:
@vstinner it was interesting for me, if "module" keyword has meaning with "export" how gcc will treat "module" without "export" keyword ?

Reply:
So far, I failed to reproduce https://bugs.python.org/issue39355 issue: using the Python C API in C++ doesn't emit any compiler warning or compiler error. See my test: https://bugs.python.org/issue39355#msg416256

Reply:
@vstinner thanks for great investigation!

Reply:
You're welcome. I'm now considering serious to convert https://github.com/python/cpython/pull/32175 to a real unit test to make sure that the Python C API is compatible with C++.

Reply:
First step: https://bugs.python.org/issue47164